### DISCUSSION TITLE
Documentation site theme choice - Please remember it!

### DISCUSSION THREAD

**Discussion by RickKukiela** [primefaces/community#233] [PrimeVue]
Is there any plan (or possible to make it a plan) to make it so that when I set a theme on the doc sites that it remembers it on subsequent page loads?

I spend a lot of time with docs open and the bright white screens kill my vibe while I'm working so I'm constantly having to waste time setting my theme preference back to what I want (dark).

I understand the main reason the theme selector is there is to preview what components will look like in the different themes, but if we at least had an option to remember this choice on future visits via a cookie or something that would be super sweet.

Thanks for coming to my TED talk.

EDIT:
Since this was moved to "PRIMEVUE" I just wanted to specify the problem is _also_ present on the "NG" and "REACT" versions of the component documentation portals:

https://primeng.org/installation/
https://primereact.org/installation/
https://primevue.org/installation/

**Comment by jepsar** [comment]
What doc sites are you talking about? Do you have a URL for us?

**Reply by RickKukiela** [reply]
All of them? The primevue documentation site for example. The official doc sites where you can get the breakdown of a component API etc.

**Reply by RickKukiela** [reply]
All of them as in this problem seems to be present on all of the documentation sites. Primevue, primeng, primefaces, etc

**Reply by jepsar** [reply]
The PrimeFaces documentation does not have a theme switch https://primefaces.github.io/primefaces/13_0_0/#/

For the other component libraries, please ask in the corresponding category.

**Reply by FlipWarthog** [reply]
I think this is in reference to the showcase itself. A similar request was made in the PrimeVue issues: https://github.com/primefaces/primevue/issues/4664

**Reply by jepsar** [reply]
In that case an issue should be opened at the PrimeFaces repo. Only I'm not sure if a fix will survive the PrimeTek redesign. I think they were working on something new. CC @melloware

**Reply by RickKukiela** [reply]
Yes sorry I guess I assumed the "primefaces" doc site had the same problem as described which is present on the "subfamily" doc sites:

https://primeng.org/installation/
https://primereact.org/installation/
https://primevue.org/installation/

I use VUE so this came up for me on that site, but I wanted to point out that the issue is present on "all" of the doc sites.

**Reply by jepsar** [reply]
I changed the category to PrimeVue to let the corresponding experts have a look.
